posthog/test/test_instance_setting_model.py

```diff
@@ -67,3 +69,22 @@ def test_can_retrieve_multiple_settings(db):
         "MATERIALIZED_COLUMNS_ENABLED": True,
         "ASYNC_MIGRATIONS_AUTO_CONTINUE": True,
     }
+
+
+@pytest.mark.parametrize("cached_value", ("true", '["1:cool","abc:123"]', '"a_string"', "123"))
```

Comment:
minor nit: distinctIDs can have `:` in them 😅 , add test to see we handle this well? for the case when someone with a `:` comes in to hammer us

Reply:
Nothing changes from a JSON deserialization perspective: the result is a string with multiple `:` in them. If the current capture endpoint can handle multiple `:` it should still handle them here.

Regardless, I'll add an extra id in the test with some `:` in it.

Reply:
yeah my intention was to test that we handle the split to figure out what key we're using correctly